Assets/MixedRealityToolkit/Inspectors/Profiles/BaseMixedRealityProfileInspector.cs

```diff
@@ -362,22 +322,14 @@ private static bool IsConcreteProfileType(String profileTypeName)
             return profileTypeName != BaseMixedRealityProfileClassName;
         }
 
-        private static void CreateStyles()
+        /// <summary>
+        /// Checks if the profile is locked and displays a warning.
+        /// </summary>
+        /// <param name="target"></param>
+        /// <param name="lockProfile"></param>
+        protected static bool CheckProfileLock(BaseMixedRealityProfile profile)
```

Comment:
>CheckProfileLock [](start = 30, length = 16)

Nit: The naming "CheckProfileLock" for bools can be ambiguous, because it's not immediately obvious what a true/false value means (i.e. does this mean "yes, we checked?", or "yes, we checked, and it's locked?). Maybe IsProfileLocked?